src/components/HiddenText/__tests__/__snapshots__/hiddenText.jest.js.snap

```diff
@@ -2,7 +2,8 @@
 
 exports[`renders correctly with empty props 1`] = `
 <span
-  className="hidden-text--wrapper"
+  className="wrapper hidden-text--wrapper"
+  data-testid="hidden-text_hiddenText"
```

Comment:
do we have conventions for data-test id? why do we use BEM?

Reply:
This is our convention now
`data-testid={dataTestId || getTestId(ELEMENT_TYPES.HIDDEN_TEXT, id)}`
But it depends on id, which can vary that's why sometimes it may look strange - need to fix ids in tests, but that's out of scope for now